packaging/installer/netdata-uninstaller.sh

```diff
@@ -733,20 +733,20 @@ rm_file /etc/cron.daily/netdata-updater
 rm_file /etc/cron.d/netdata-updater
 
 
-if [ -n "${NETDATA_PREFIX}" ] && [ -d "${NETDATA_PREFIX}" ]; then
+if [ -n "${NETDATA_PREFIX}" ] && [ -d "${NETDATA_PREFIX}" ] && [ "netdata" = "$(basename "$NETDATA_PREFIX")" ] ; then
   rm_dir "${NETDATA_PREFIX}"
 else
-  rm_file "/usr/sbin/netdata"
-  rm_file "/usr/sbin/netdatacli"
+  rm_file "${NETDATA_PREFIX}/usr/sbin/netdata"
```

Comment:
The only thing that confuses me is the `[ -d "${NETDATA_PREFIX}" ]` check. Why do we need it?

If we need it and it is false, should we not use `${NETDATA_PREFIX}` in the `else` branch?

Reply:
rm dir you execute only  if it's a dir. 

I left `-n` hoping that it wont make the third check fail (by short-circuiting the expression) but to be honest it cant fail

Reply:
> rm dir you execute only if it's a dir.

I know, my question was about the `[ -d "${NETDATA_PREFIX}" ]` check and whether we need to use ${NETDATA_PREFIX} in the else branch if the check fails.